testsuite/forge/src/backend/k8s/prometheus.rs

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::Result;
+
+use anyhow::bail;
+use aptos_logger::info;
+use prometheus_http_query::{response::PromqlResult, Client as PromClient};
+use std::{collections::BTreeMap, convert::TryFrom};
+
+pub fn get_prometheus_client() -> Result<PromClient> {
```

Comment:
nit: Let's be consistent with the naming. I see that we are using both `Prom` or `Prometheus`. I would prefer if we use `Prometheus` instead of `Prom` for clarity. So we can rename `PromClient` to `PrometheusClient` ?